intelmq/bin/intelmqctl.py

```diff
@@ -445,7 +457,7 @@ def __init__(self, interactive: bool = False, return_type: str = "python", quiet
             subparsers = parser.add_subparsers(title='subcommands')
 
             parser_list = subparsers.add_parser('list', help='Listing bots or queues')
-            parser_list.add_argument('kind', choices=['bots', 'queues'])
+            parser_list.add_argument('kind', choices=['bots', 'queues', 'queues-and-status'])
```

Comment:
Could you please add that to `bash-completion/intelmq.sh` line 40?

Reply:
thanks for introducing me into that; I've also added some missing completions in there